requirements.txt

```diff
@@ -5,3 +5,4 @@ pbr
 termcolor==1.1.0
 setproctitle
 ordereddict==1.1
+ujson
```

Comment:
Seems not in centos official repo list and it's in debian testing, so not packaged :/

Perhaps do this dependency optional and in code use ujson if module installed